src/iperf_api.h

```diff
@@ -474,7 +474,8 @@ enum {
     IEPTHREADCANCEL=151,        // Unable to cancel thread (check perror)
     IEPTHREADJOIN=152,		// Unable to join thread (check perror)
     IEPTHREADATTRINIT=153,      // Unable to initialize thread attribute (check perror)
-    IEPTHREADATTRDESTROY=154,      // Unable to destroy thread attribute (check perror)
+    IEPTHREADSIGMASK=154,      // Unable to initialize sub thread signal mask (check perror)
+    IEPTHREADATTRDESTROY=155,      // Unable to destroy thread attribute (check perror)
```

Comment:
Could you leave `IEPTHREADATTRDESTROY` with its original numerical value of 154, and just add `IEPTHREADSIGMASK` as 155? It probably doesn't make a huge difference but it's probably better to not change existing definitions if there's not a reason to.

Reply:
Sure, there will be a push later after local compilation and testing.